src/components/App.js

```diff
@@ -0,0 +1,56 @@
+import { getUser, getFolder } from "../api";
+import "./App.css";
+import NavigationBar from "./NavigationBar";
+import { useEffect, useState } from "react";
+import FolderBar from "./FolderBar";
+import CardSection from "./CardSection";
+import Footer from "./Footer";
+
+function App() {
+  const [profile, setProfile] = useState(null);
+  const [folderName, setFolderName] = useState("");
+  const [userName, setUserName] = useState("");
+  const [profileImage, setProfileImage] = useState("");
+  const [cardList, setCardList] = useState(null);
+
+  useEffect(() => {
+    async function getProFile() {
+      const user = await getUser();
+      if (user) {
+        const { email, profileImageSource } = user;
+        setProfile({ email, profileImageSource });
```

Comment:
구조분해할당을 해주지 않고 user를 바로 넣어도 동일할 것 같아요!
```suggestion
        setProfile(user);
```